Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LightSensorSamplePage.xaml

```diff
@@ -0,0 +1,71 @@
+﻿<Page
+	x:Class="Uno.Gallery.Views.Samples.LightSensorSamplePage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:Uno.Gallery"
+	xmlns:samples="using:Uno.Gallery.Views.Samples"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+	xmlns:smtx="using:ShowMeTheXAML"
+	mc:Ignorable="d">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout>
+
+			<local:SamplePageLayout.FluentTemplate>
+				<DataTemplate>
+					<StackPanel>
+						<smtx:XamlDisplay UniqueKey="LightSensorSamplePage_Sample"
+										  smtx:XamlDisplayExtensions.IgnorePath="XamlDisplay\StackPanel"
+										  smtx:XamlDisplayExtensions.Header="Light Sensor">
+							<StackPanel Spacing="20">
+								<!-- C# code
+var lightSensor = LightSensor.GetDefault();
+			
+if (lightSensor != null)
+{
+	lightSensor.ReadingChanged += Accelerometer_ReadingChanged;
+}
+else
+{
+	//LightSensor is not available on this platform/device.
+}
+
+private void LightSensor_ReadingChanged(LightSensor sender, LightSensorReadingChangedEventArgs args)
+{
+	LastReadTimestamp = args.Reading.Timestamp;
+	IlluminanceInLux = args.Reading.IlluminanceInLux;
+}
+-->
+								
+								<StackPanel.DataContext>
+									<samples:LightSensorSamplePageViewModel />
+								</StackPanel.DataContext>
```

Comment:
FYI the SamplePageAttribute has a DataType property that will set the DataContext of the sample for you. Similar to this: https://github.com/unoplatform/Uno.Gallery/blob/7b37b225839b45d74e931f772c99760986d32f7a/Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/TabBarSamplePage.xaml.cs#L26

Reply:
I did not know that - but it is also not working. I removed the DataContext-XAML on the Page and added the DataType-property on the SamplePageAttribute. Then I get Binding errors. What do I do wrong?

Reply:
I think the constructor of the `ViewModel` must have a `Dispatcher` as parameter, but as long as the current implementation works, I think it is fine

Reply:
Hm, the TabBarViewModel also does not have a `Dispatcher` a s parameter.
I'll leave it how it is. Feel free to come back if you want me to change this. I'm happy to be in line with the best practice here, but I don't see my error.